### DISCUSSION TITLE
Update any Date module for Nuxt 3 (momentjs, date-fns, day.js)

### DISCUSSION THREAD

**Discussion by mrleblanc101** [nuxt/nuxt#16077] [Questions]
None of the 3 majors Date modules support Nuxt 3.
https://www.npmjs.com/package/@nuxtjs/date-fns
https://www.npmjs.com/package/@nuxtjs/dayjs
https://www.npmjs.com/package/@nuxtjs/moment

Is there a plan to update any of them ?

**Comment by wobsoriano** [comment]
Someone tried dayjs I guess https://github.com/nuxt-community/dayjs-module/issues/376

**Reply by mrleblanc101** [reply]
I was able to use vanilla `moment` (not the `@nuxtsj/moment` version) with `moment-duration-format` by creating a custom plugin but it load all locales by default as moment is not tree-shakable.

I wouldn't know how to strip them as Nuxt 3 use Vite and not Webpack and `@nuxtsj/moment` used a Webpack module to achieve this.

Otherwise it work great.

**Reply by wobsoriano** [reply]
Also, https://github.com/Dunqing/unplugin-moment-to-dayjs
